phpBB/phpbb/db/migration/data/v33x/bot_update_vol2.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v33x;
+
+class bot_update extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return ['\phpbb\db\migration\data\v330\v330'];
+	}
```

Comment:
`return ['\phpbb\db\migration\data\v33x\v334'];`

Reply:
It does not "depend" on the migration step done before in v3.3.x, but probably good to have things in a clear order. Change applied.